src/redux/books/booksSlice.js

```diff
@@ -1,8 +1,28 @@
 /* eslint-disable no-param-reassign */
```

Comment:
- [ ] it would be better to remove the eslint disable because it ensures that you write clean code

Reply:
thanks for your review @youmari